builtin/logical/pki/cert_util.go

```diff
@@ -571,6 +572,20 @@ func generateCreationBundle(b *backend,
 		}
 	}
 
+	// Set OU (organizationalUnit) values if specified in the role
+	ou := []string{}
+	{
+		if role.OU != "" {
```

Comment:
I was thinking about that too; I realized that elsewhere in PKI we are doing this parsing every time, *but* that's because much of it was written around the 0.2.1 era.

When the role is read back via a GET it should come back with an array, so storing it as one is probably a good idea as you suggest.